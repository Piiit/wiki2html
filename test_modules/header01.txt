~~NOTOC~~
==first line==
before
=header=
this is a header.

== header with trailing whitespaces        ==

== header with trailing equal signs ====
