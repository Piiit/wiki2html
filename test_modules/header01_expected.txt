<h2>first line</h2>
before
<h1>header</h1>
this is a header.

<h2>header with trailing whitespaces</h2>

<h2>header with trailing equal signs ==</h2>
